### ISSUE TITLE
Please consider single-line comment JavaScript inlining

### ISSUE THREAD

**Issue by TriEdgeAI** [bridgedotnet/Bridge#2113]
The JavaScript inlining with `/*@ alert("hello") */` is really nice.
It would also be nice to have `//@ alert("hello");`.

[Deck](http://deck.net/28a0f1181da575430156eb079414cc3b)
```cs
public class Program
{
    public static void Main()
    {
        /*@ alert("Multiline inline comment") */
        //@ alert("Single inline comment");
    }
}
```

**Comment by geoffreymcgill** [comment]
Nice suggestion. Should be easy to implement.

**Comment by geoffreymcgill** [comment]
Pull-request #2125 has been submitted with an implementation of this feature request. 

**Input (.cs)**

```cs
//@ var tmp1;
//@ var  tmp2;
//@ var tmp3; // test
//@ var tmp4; //@ test
```

**Output (.js)**

```js
var tmp1;
var  tmp2;
var tmp3; // test
var tmp4; //@ test
```

Once the pull-request has been reviewed, this feature should be included in the upcoming Bridge 15.5 release. 

Thanks again for the suggestion.

**Comment by geoffreymcgill** [comment]
Bridge 15.5 has been released, and now includes this inline commenting functionality.

http://deck.net/e369180d2851571bfaa6d45dbdf29e86

Hope this helps.
